bigtable/admin.go

```diff
@@ -1457,10 +1457,17 @@ func UpdateInstanceAndSyncClusters(ctx context.Context, iac *InstanceAdminClient
 
 // RestoreTable creates a table from a backup. The table will be created in the same cluster as the backup.
 func (ac *AdminClient) RestoreTable(ctx context.Context, table, cluster, backup string) error {
+	return ac.RestoreTableTo(ctx, ac.instance, table, cluster, backup)
+}
+
+// RestoreTableTo creates a new table by restoring from this completed backup.
+//
+// diffInstance is an instance in which the new table will be restored to.
+// Instance must be in the same project as the project containing backup.
+func (ac *AdminClient) RestoreTableTo(ctx context.Context, diffInstance, table, cluster, backup string) error {
```

Comment:
Can you rename params to something like `dst` for the destination? What do other languages use? Also I think the destination should probably be the last param rather than the first.

Docs can also be clarified/cleaned up a bit to highlight difference from `RestoreTable`.